crates/bevy_app/src/app_builder.rs

```diff
@@ -273,13 +427,44 @@ impl AppBuilder {
         self
     }
 
+    /// Adds a group of plugins
+    ///
+    /// Bevy plugins can be grouped into a set of plugins. By default
+    /// bevy provides a few lists of plugins that can be used to kickstart
```

Comment:
Nit: "By default bevy provides a few lists of plugins that can be used to kickstart the development." -> "Bevy provides built-in PluginGroups that provide core engine functionality"

Reply:
Fixed